assets/js/apitool-methods-ws.js

```diff
@@ -581,6 +581,15 @@ Request('fee', {
   }
 })
 
+Request('server_definitions', {
+  description: "Returns an SDK-compatible `definitions.json`, generated from the rippled server being queried.",
```

Comment:
I'm pretty sure the description isn't parsed as Markdown, so this will just show backticks to the end-user. (Disclaimer: I didn't test to confirm)

Simplest solution is just to go without formatting.

```suggestion
  description: "Returns an SDK-compatible definitions.json, generated from the rippled server being queried.",
```